### PR TITLE
fix(components/TDP-9660): lint errors RatioBar/QualityBar

### PR DESCRIPTION
**Pull request by trob-tlnd**
**What is the problem this PR is trying to solve?**

#3140 introduced lint errors.

**What is the chosen solution to this problem?**

fix them all

**Please check if the PR fulfills these requirements**

* [ ] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

**Review by jmfrancois — APPROVED** [review]

File: packages/components/src/QualityBar/QualityBar.component.js [hunk 1]
```diff
@@ -53,27 +53,42 @@ export function QualityBar({ invalid, valid, empty, onClick, getDataFeature, dig
...
-					type: QualityType.INVALID
-				}) : null}
+				onClick={
+					onClick
+						? e =>
+								onClick(e, {
+									type: QualityType.INVALID,
+								})
+						: null
+				}
```
Comment (romainseb) [inline packages/components/src/QualityBar/QualityBar.component.js:63]: Is there a formating issue ? It's looking weird to me

**Review by romainseb — COMMENTED** [review]
Formating seems weird for me

File: packages/components/src/QualityBar/QualityBar.component.js (hunk 1 above)
Comment (trob-tlnd) [reply packages/components/src/QualityBar/QualityBar.component.js:63]: problem of linter ?

**Review by romainseb — APPROVED** [review]

File: packages/components/src/QualityBar/QualityBar.component.js (hunk 1 above)
Comment (romainseb) [reply packages/components/src/QualityBar/QualityBar.component.js:63]: Forget about it, looks good now, it was the code that was not formatted well before

**Pull request merged**